### ISSUE TITLE
Unable to execute sqlite3 queries, change the sqlite.sqlite3 setting to fix this issue.

### ISSUE THREAD

**Issue by alexandlazaris** [AlexCovizzi/vscode-sqlite#40]
Simply as title says. 

1. Installed the VSCode extension
2. Selected command >SQLite: Open Database in Explorer
3. Error window in bottom right with title message

![image](https://user-images.githubusercontent.com/17981265/48768764-1e641280-ed0e-11e8-96c5-d39b3d7e0b28.png)

Is this just me? I'm on Win10. Additional VSCode details below:

Version: 1.29.1 (system setup)
Commit: bc24f98b5f70467bc689abf41cc5550ca637088e
Date: 2018-11-15T19:13:36.375Z
Electron: 2.0.12
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64

**Comment by AlexCovizzi** [comment]
The extension needs a sqlite3 executable to run the queries, it will first look in the ```sqlite.sqlite3``` setting, if this doesn't work (the command or path provided is not a valid ```sqlite3``` executable), it will fallback to the binary in the ```[extension path]/bin``` folder, if this doesn't work either it will show that error.

To fix the problem you need to provide a valid path or command for the sqlite3 executable in the ```sqlite.sqlite3``` setting.

**Comment by alexandlazaris** [comment]
No luck for me. I've added variations of the path of "sqlite-win32-x86" however I still get the failure message in the bottom right. 

This is my latest field value:
![image](https://user-images.githubusercontent.com/17981265/48984666-c33d8000-f152-11e8-8d40-4857008e746d.png)

I can execute the .exe through CLI however ... (if that helps)

**Comment by AlexCovizzi** [comment]
Can you post here the output you get in OUTPUT > SQLite

![cattura](https://user-images.githubusercontent.com/16863733/49080359-a8970400-f243-11e8-9bd5-fabad0acab9b.PNG)

**Comment by MrSinamun** [comment]
I've had the same issue.

Here is my output:
```
[7:27:16 PM][vscode-sqlite][INFO] Activating extension vscode-sqlite v0.3.2...
[7:27:16 PM][vscode-sqlite][WARN] 'C:\sqlite\sqlite3.exe' is not recognized as a command.
[7:27:16 PM][vscode-sqlite][INFO] Fallback binaries found: 'c:\Users\USER NAME\.vscode\extensions\alexcvzz.vscode-sqlite-0.3.2\bin\sqlite-win32-x86.exe'
[7:27:16 PM][vscode-sqlite][ERROR] Command failed: c:\Users\USER NAME\.vscode\extensions\alexcvzz.vscode-sqlite-0.3.2\bin\sqlite-win32-x86.exe -version
'c:\Users\USER NAME' is not recognized as an internal or external command,
operable program or batch file.
```

So it does not recognize the path to my sqlite3.exe as a command. If I however paste this path into my terminal it starts without any issues:

```
C:\Users\USER NAME>C:\sqlite\sqlite3.exe
SQLite version 3.26.0 2018-12-01 12:34:55
Enter ".help" for usage hints.
Connected to a transient in-memory database.
Use ".open FILENAME" to reopen on a persistent database.
sqlite>
```

I have a username with a whitespace which seems to be why the fallback does not work either.

I have the user setup version of vscode:
```
Version: 1.29.1 (user setup)
Commit: bc24f98b5f70467bc689abf41cc5550ca637088e
Date: 2018-11-15T19:13:36.375Z
Electron: 2.0.12
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64
```

**Comment by AlexCovizzi** [comment]
Thanks for your help @MrSinamun, I will try to publish a fix as soon as possible, it could take while though, since this next weeks i'm a little busy.

**Comment by alexandlazaris** [comment]
Btw @AlexCovizzi , is this version of sqlite safe from this recently published security flaw? https://cybersecurityreviews.net/2018/12/16/critical-sqlite-flaw-leaves-millions-of-apps-vulnerable-to-hackers/

**Comment by AlexCovizzi** [comment]
The bug is fixed in the new version 0.3.3, i also updated the executables to the new version 3.26.0.
